en-US/Punctuation.xml

```diff
@@ -107,15 +107,53 @@
 							magenta
 						</para>
 
-					</listitem>
+					</listitem> 
 
-				</itemizedlist>
+				</itemizedlist> 
+
+			</listitem>  -->
+
+		</itemizedlist> 
+
+		 <para>
+			Use a semicolon to separate items in a series if the items contain commas.
+		</para>
+		 <itemizedlist>
+			<listitem>
+				<para>
+					Everyday I have coffee, toast, and fruit for breakfast; a salad for lunch; and a peanut butter sandwich, cookies, ice cream, and chocolate cake for dinner.
+				</para>
 
 			</listitem>
 
 		</itemizedlist>
 		 <para>
-			Use a colon after "as follows" and "the following" if the related list immediately follows the stem, or introductory sentence.
+			Use a semicolon before a conjunctive adverb, such as <emphasis>however, therefore, otherwise, namely, for example</emphasis>, and so on.
```

Comment:
I suggest clarifying that the semicolon is only used with conjunctive adverbs in the case of two independent clauses -- otherwise, using commas would be the norm. (Readers might misinterpret this guidance to mean that all instances of conjunctive adverbs require semicolons.) Suggested revision: "When a conjunctive adverb (such as <emphasis>however, therefore, otherwise, namely, for example</emphasis>, and so on) joins two independent clauses, use a semicolon before the conjunctive adverb and a comma after it." 

To parallel this guidance, consider adding a corresponding section in the "2.8.2. Commas" section > "With conjunctive adverbs": "When using a conjunctive adverb (such as <emphasis>however, therefore, otherwise, namely, for example</emphasis>, and so on) in a sentence, set it off with commas. Example1: "It rained all afternoon. As a result, we canceled our picnic." Example2: "The grass, however, stayed dry under the trees." (The examples are meant to show adjunctive adverbs in different places in the sentence. This guidance would be good to include because this is a somewhat common error.)

Note: I used the emphasis tag as in the original, but it's not rendering correctly in this comment.